src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1383,6 +1382,8 @@ def forward(
                 text_model_last_hidden_states,
                 vision_model_last_hidden_states,
             )
+            output = tuple(x for x in output if x is not None)
```

Comment:
We need to remove the `None` elements from the outputs if `return_dict=False`.
This is what we do for all models.
And it is also necessary for `jit.trace`

Reply:
Thank you, ran the tests and everything looks good!